packages/component-library/components/Spacing/padding.ts

```diff
@@ -0,0 +1,77 @@
+import { GridFractions, ResponsiveSpacing, Spacing } from "../types"
+import { convertFractionToString } from "./util"
+
+const styles = require("./Padding.module.scss")
+
+export const pt = (unit: GridFractions): string[] => [
+  styles[`pt-${convertFractionToString(unit)}`],
+]
+
+export const pr = (unit: GridFractions): string[] => [
+  styles[`pr-${convertFractionToString(unit)}`],
+]
+
+export const pb = (unit: GridFractions): string[] => [
+  styles[`pb-${convertFractionToString(unit)}`],
+]
+
+export const pl = (unit: GridFractions): string[] => [
+  styles[`pl-${convertFractionToString(unit)}`],
+]
+
+export const px = (unit: GridFractions): string[] => [
+  styles[`px-${convertFractionToString(unit)}`],
+]
+
+export const py = (unit: GridFractions): string[] => [
+  styles[`py-${convertFractionToString(unit)}`],
+]
+
+export const p = (unit: ResponsiveSpacing): string[] => {
+  const classes: string[] = []
+  if (typeof unit === "string") {
+    classes.push(styles[`p-${convertFractionToString(unit)}`])
+  } else {
+    Object.keys(unit).forEach(key => {
+      classes.push(styles[`p-${convertFractionToString(unit[key])}--${key}`])
+    })
+  }
+
+  return classes
+}
+
+// @TODO - i18n. Use padding before / padding after
+// Seb can help with this
+export const responsivePaddingClasses = ({
```

Comment:
Hey @sebpearce could you help with this 👼? Do we switch paddings and margins for LTR languages?

Reply:
https://github.com/cultureamp/kaizen-design-system/blob/master/packages/component-library/styles/layout.scss

Yep we actually have mixins for this:

![image](https://user-images.githubusercontent.com/6406263/74794298-ef748c00-5316-11ea-97d5-09be78db3904.png)

How we do this in these components is up for debate.

Reply:
Interesting... That throws a spanner in the works! So looking through the repo and this applied in certain contexts and it shouldn't be a global rule. With that in mind here are some initial thoughts 

Box could take a rtl prop. When this is applied all left and right paddings and margins will swap. Where this is slightly annoying is it will apply to both margin and padding, and developers may only want rtl on one of those. To mitigate they could use another box without `rtl`.
```
<Box px="2">
   <Box rtl={true} mr="4" ml="2>Content</Box>
</Box>
// outputs
margin-right: 2*$ca-grid;
margin-left: 4*$ca-grid;
padding: 2*$grid 0;
```

Thoughts?

Reply:
^ This is what I've implemented. If someone thinks this is a bad idea let me know!

Reply:
RTL prop sounds 👌 to me. We're doing this already in a few cases I think.